content/gophergala/data-mining.md

```diff
@@ -0,0 +1,154 @@
++++
+author = ["Marcel Hauf"]
+date = "2015-02-02T17:10:37+01:00"
+linktitle = "data-mining"
+title = "Data mining with Go"
++++
+
+# What is the idea behind Gophergala project heatingeffect
+
+The idea is to provide a visual representation of the raw Notice data from chillingeffects.org.
+This is achieved by getting the data from chillingeffects.org via a worker called harvester.
+The harvested data gets stored directly in a MongoDB database.
+A second worker aggregates the harvested data and stores it in the MongoDB database.
+The aggregated data is displayed through a simple Go http server.
+
+This post will focus on how data gets from chillingeffects.org into a MongoDB database via the harvester.
+
+## [chillingeffects.org](https://chillingeffects.org/)
+
+> The Chilling Effects database collects and analyzes legal complaints and requests for removal of online materials, helping Internet users to know their rights and understand the law. 
```

Comment:
Suggestion: Move this up right after mentioning chillingeffects.org so people know what it is.

Then get into the technical details of harvester and MongoDB after that.